CONTRIBUTING.rst

```diff
@@ -6,15 +6,15 @@ What to work on?
 ----------------
 
 You are welcome to propose and contribute new ideas.
-We encourage you to `open an issue <https://github.com/simai-ml/MAPIE/issues>`_ so that we can align on the work to be done.
+We encourage you to `open an issue <https://github.com/scikit-learn-contrib/MAPIE/issues>`so that we can align on the work to be done.
 It is generally a good idea to have a quick discussion before opening a pull request that is potentially out-of-scope.
 
 Fork/clone/pull
 ---------------
 
 The typical workflow for contributing to `mapie` is:
 
-1. Fork the `master` branch from the `GitHub repository <https://github.com/simai-ml/MAPIE>`_.
+1. Fork the `master` branch from the `GitHub repository <github.com/scikit-learn-contrib/MAPIE>`_.
```

Comment:
Why did we remove the https://?

Reply:
Yes, indeed, well spotted. Thank you. I just corrected it